src/main/java/nicole/taskstorage/TaskList.java

```diff
@@ -0,0 +1,189 @@
+package nicole.taskstorage;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+import nicole.nicoleexceptions.NicoleException;
+import nicole.task.Event;
+import nicole.task.Task;
+
+public class TaskList {
+    protected static final List<Task> TASKS = new ArrayList<>();
```

Comment:
Is there a specific reason why we should use the keywords "static final" here? It is true that there can only be one list which makes it a constant, but the list is modifiable by adding and deleting elements, which makes the use of keywords a bit unintuitive to me.